### DISCUSSION TITLE
[PROD SUPPORT] Eng features that can aid migration of websites

### DISCUSSION THREAD

**Discussion by seaerchin** [isomerpages/isomercms-frontend#886] [Ideas]
tagging @jacksonOGP for ideas

1. bulk upload of files/images
2. paste html into text editor (@seaerchin will look into this next week)  
    - thinking about using the env to determine if the feature is active/have a checkbox???

feel free to chime in with any ideas and we'll create issues. doing the github discussion here so that it's tracked; will need PM guidance on prioritisation but will take a week to look into the html copy paste one and this should proceed concurrently with FE migration.

**Comment by jacksonOGP** [comment]
1. I understand there is some discussion on bulk files/images but what is the reason why we couldn't upload copy and paste a single image into the editor? Github supports this afaik (at least in this comment section we're seeing)


2. no problem w this

**Reply by seaerchin** [reply]
i think if we do #2, we might be able to get what you described in your first point but it'll point to the original image url rather than an isomer hosted one

not sure if we can auto upload etc cos of naming conflicts etc etc (if we can take a hash of unix timestamp + image name, it should be ok but need more input here)
